articles/azure-functions/functions-bindings-service-bus.md

```diff
@@ -109,6 +111,8 @@ If you have `isSessionsEnabled` set to `true`, the `sessionHandlerOptions` is ho
 |Property  |Default | Description |
 |---------|---------|---------|
 |prefetchCount|0|Gets or sets the number of messages that the message receiver can simultaneously request.|
+| transportType| amqpTcp | The protocol and transport that is used for communicating with Service Bus. Available options: `amqpTcp`, `amqpWebSockets`|
```

Comment:
This is documented under the wrong extension.

Reply:
Whoops.    Got the wrong table there.